src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -110,6 +111,21 @@ public void add(Person toAdd) throws DuplicatePersonException {
         internalList.add(toAdd);
     }
 
+    /**
+     * sort the addressbook list by name's alphabetical order
+     */
+    public void sort(){
+         Collections.sort(internalList,
+
+ //               (personOne,personTwo) -> personOne.getName().toString().compareToIgnoreCase(personTwo.getName().toString()));
```

Comment:
You should remove unnecessary code from your PR, since with git, you can always trace back to previous versions of the code.

Reply:
Thanks for your suggestions.